dallinger/command_line.py

```diff
@@ -171,6 +172,14 @@ def verify_package(verbose=True):
             "{0} to {1}.".format(base_pay, dollarFormat), delay=0, chevrons=False, verbose=verbose)
         is_passing = False
 
+    # Validate qualification blacklist config
```

Comment:
That was my first impulse, but I worried about making it easy to set up aspects of a HIT implicitly/non-deliberately. I trust your instincts here much more than my own, so if you think that's OK let's do that.